multiapps-controller-core/src/main/java/org/cloudfoundry/multiapps/controller/core/validators/parameters/RoutesValidator.java

```diff
@@ -26,7 +28,7 @@ public RoutesValidator(String namespace, boolean applyNamespaceGlobal) {
 
     protected void initRoutesValidators(String namespace, boolean applyNamespaceGlobal) {
         ParameterValidator routeValidator = new RouteValidator(namespace, applyNamespaceGlobal);
-        this.validators = Map.of(routeValidator.getParameterName(), routeValidator);
+        this.validators = Map.of(routeValidator.getParameterName(), routeValidator, LIVE_ROUTE, routeValidator);
```

Comment:
Why the second validator with a different parameter name key? Do we need to run the validation again for our cosntructed 'live' parameters?

Reply:
if we don't add the live parameter key here, the validation part of the code will remove it from the map